manifests/a/Ascension/AscensionLauncher/1.79/Ascension.AscensionLauncher.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Ascension.AscensionLauncher
+PackageVersion: 1.79
+PackageLocale: en-US
+Publisher: Ascension

```

```suggestion
Publisher: Project Ascension
```
This must match Control Panel.